templates/operator/params.yaml.template

```diff
@@ -273,6 +273,10 @@ parameters:
     description: "Authentication backend, implementing IAuthenticator; used to identify users."
     default: "AllowAllAuthenticator"
 
+  - name: AUTHENTICATION_CREDENTIALS_SECRET
```

Comment:
To keep the parameter names consistent we might want to name this "AUTHENTICATION_SECRET_NAME" to be in sync with "TLS_SECRET_NAME"